src/components/side_panel/components/checkbox/checkbox.xml

```diff
@@ -2,6 +2,7 @@
   <t t-name="o-spreadsheet.Checkbox">
     <label
       class="o-checkbox d-flex align-items-center"
+      role="button"
```

Comment:
![image](https://github.com/user-attachments/assets/06383e2e-08a4-46fe-91ad-9e7c0ab510bc)

Why is it now necessary ? :eyes:

Reply:
To change the pointer to indicate that the label is clickable

Reply:
I edited the commit to have the cursor: pointer also applying to the input (some other rules interfered with it, namely the appearance:none and its variants)